packages/perspective/src/js/api/client.js

```diff
@@ -65,6 +65,23 @@ export class Client {
         }
     }
 
+    async memory_usage() {
+        return await new Promise((resolve, reject) => {
+            ++this._worker.msg_id;
```

Comment:
Seems like the incrementing msg_id, setting a handler etc. logic here is duplicated from `post` - seems like you could probably just call `post({cmd: "memory_usage"}, resolve, reject)` here